BUILD.gn

```diff
@@ -124,3 +119,20 @@ group("dist") {
     "$flutter_root/sky/dist",
   ]
 }
+
+# Fuchsia currently only supports a subset of our unit tests
+# When adding a new dep here, please also ensure the dep is added to
+# testing/fuchsia/run_tests.sh and testing/fuchsia/test_fars
+if (is_fuchsia) {
+  group("fuchsia_tests") {
+    testonly = true
+
+    deps = [
+      "$flutter_root/flow:flow_tests",
```

Comment:
You will need to remove this dep as this branch doesn't have the flow_tests added.

Reply:
Done